src/pilot/assign_role.py

```diff
@@ -405,6 +405,13 @@ def check_cntlr_physical_disks_len(cntrl_physical_disks):
         return True
 
 
+def _fetch_boss_controller(drac_client, raid_controllers):
+    boss_controller = [cntrl for cntrl in raid_controllers
+                       if drac_client.is_boss_controller(cntrl)]
+    if boss_controller:
+        return boss_controller[0]
```

Comment:
@gaelrehault and @heydog15, what should be done if there is more than 1 BOSS found in the system? Is it OK to return the first one?

Reply:
@gaelrehault, confirmed that 2 BOSS cards is not valid configuration.